ios/Classes/FlutterBleLibPlugin.m

```diff
@@ -158,7 +160,14 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
 
 // MARK: - MBA Methods - BleClient lifecycle
 
+- (void)isClientCreated:(FlutterMethodCall *)call result:(FlutterResult)result {
+    result([NSNumber numberWithBool:_adapter != nil]);
```

Comment:
this could just as well be simply `result(@(adapter != nil))`
as there is a literal way to create NSNumber instances 
`@(int|float|boolean)` == [NSNumber numberWith[Int,Float,Bool]:(int|float|boolean).
Similarly to the literal to create NSStrings `@""`

Reply:
Fun fact: they're apparently not the same 😄 
```objc
result(@(_adapter != nil))
```
gives:
> type 'int' is not a subtype of type 'bool?' in type cast
While using `NSNumber....` works. 🤷

Reply:
More info: using literal notation sends 0 to Dart, using NSNumber numberWithBool returns bool.

Reply:
this is really odd.. https://clang.llvm.org/docs/ObjectiveCLiterals.html